baaas-decision-fleet-manager/src/main/java/org/kie/baaas/dfm/app/model/Decision.java

```diff
@@ -37,7 +37,8 @@
 
 @NamedQueries({
         @NamedQuery(name = "Decision.byCustomerIdAndName", query = "from Decision where customerId=:customerId and name=:name"),
-        @NamedQuery(name = "Decision.byCustomerAndIdOrName", query = "from Decision where customerId=:customerId and name=:idOrName or id=:idOrName")
+        @NamedQuery(name = "Decision.byCustomerAndIdOrName", query = "from Decision where customerId=:customerId and name=:idOrName or id=:idOrName"),
```

Comment:
A comment not related to this PR.. I think we have a bug here because this query is going to be parsed like 
`from Decision where (customerId=:customerId and name=:idOrName) or id=:idOrName"`

Reply:
I'll fix it in another PR

Reply:
sounds good